api/src/http.rs

```diff
@@ -399,8 +399,8 @@ pub fn start_http_thread(
     let socket_path = PathBuf::from(path);
 
     let mut pool = Poll::new()?;
-    let waker = Waker::new(pool.registry(), EXIT_TOKEN)?;
-    let waker = Arc::new(waker);
+    let waker = Arc::new(Waker::new(pool.registry(), EXIT_TOKEN)?);
+    let _waker = waker.clone();
```

Comment:
Should we rename it to wake2 to keep consistence with #385 ?

Reply:
I'm afraid it has to be called "_waker..." at least, since it seems unused.

Reply:
updated.

Reply:
There's a `drop(wake2)`.

Reply:
Yeah, it's an explicit use, but not sure if static analyzer changes the semantics in the future.
Anyway, I've renamed it to waker2.